### ISSUE TITLE
cudnn

### ISSUE THREAD

**Issue by zyxcambridge** [tensorflow/tensorflow#16004]
INFO:tensorflow:Starting Queues.
2018-01-10 18:45:29.789178: E tensorflow/stream_executor/cuda/cuda_blas.cc:366] failed to create cublas handle: CUBLAS_STATUS_NOT_INITIALIZED
2018-01-10 18:45:29.843774: E tensorflow/stream_executor/cuda/cuda_blas.cc:366] failed to create cublas handle: CUBLAS_STATUS_NOT_INITIALIZED
INFO:tensorflow:global_step/sec: 0
2018-01-10 18:45:33.552377: E tensorflow/stream_executor/cuda/cuda_blas.cc:366] failed to create cublas handle: CUBLAS_STATUS_NOT_INITIALIZED
2018-01-10 18:45:33.587139: E tensorflow/stream_executor/cuda/cuda_blas.cc:366] failed to create cublas handle: CUBLAS_STATUS_NOT_INITIALIZED
2018-01-10 18:45:33.621270: E tensorflow/stream_executor/cuda/cuda_blas.cc:366] failed to create cublas handle: CUBLAS_STATUS_NOT_INITIALIZED
2018-01-10 18:45:33.660148: E tensorflow/stream_executor/cuda/cuda_blas.cc:366] failed to create cublas handle: CUBLAS_STATUS_NOT_INITIALIZED
2018-01-10 18:45:34.072946: E tensorflow/stream_executor/cuda/cuda_blas.cc:366] failed to create cublas handle: CUBLAS_STATUS_NOT_INITIALIZED
2018-01-10 18:45:34.244066: E tensorflow/stream_executor/cuda/cuda_blas.cc:366] failed to create cublas handle: CUBLAS_STATUS_NOT_INITIALIZED
2018-01-10 18:45:37.678240: E tensorflow/stream_executor/cuda/cuda_dnn.cc:385] could not create cudnn handle: CUDNN_STATUS_INTERNAL_ERROR
2018-01-10 18:45:37.678276: E tensorflow/stream_executor/cuda/cuda_dnn.cc:352] could not destroy cudnn handle: CUDNN_STATUS_BAD_PARAM
2018-01-10 18:45:37.678313: F tensorflow/core/kernels/conv_ops.cc:667] Check failed: stream->parent()->GetConvolveAlgorithms( conv_parameters.ShouldIncludeWinogradNonfusedAlgo<T>(), &algorithms)

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce

**Comment by tensorflowbutler** [comment]
Nagging Awaiting Response: It has been 14 days with no activityand the `awaiting response` label was assigned. Is this still an issue?

**Comment by tensorflowbutler** [comment]
Nagging Awaiting Response: It has been 14 days with no activityand the `awaiting response` label was assigned. Is this still an issue?

**Comment by tensorflowbutler** [comment]
Nagging Awaiting Response: It has been 14 days with no activityand the `awaiting response` label was assigned. Is this still an issue?

**Comment by tensorflowbutler** [comment]
Nagging Awaiting Response: It has been 14 days with no activityand the `awaiting response` label was assigned. Is this still an issue?

**Comment by burakekincioglu** [comment]
hi have same problem ..
2018-03-20 18:03:26.184957: E tensorflow/stream_executor/cuda/cuda_dnn.cc:371] could not create cudnn handle: CUDNN_STATUS_INTERNAL_ERROR
2018-03-20 18:03:26.185004: E tensorflow/stream_executor/cuda/cuda_dnn.cc:338] could not destroy cudnn handle: CUDNN_STATUS_BAD_PARAM

how can i solve this problem

**Comment by fatmausta** [comment]
Hello,
I have the same error:
My Python crushes when it gets to the first convolutional layer, I tried making the model simpler, still crushes. 

This is the output and error: 

Do I need to update my cudnn? On  tensorflow webiste, it seems cudnn 6 still works with tensorflow-gpu 1.7

TensorFlow 1.7 may be the last time we support cuDNN versions below 6.0.
Starting with TensorFlow 1.8 release, 6.0 will be the minimum supported
version. [here](url): https://github.com/tensorflow/tensorflow/releases

My purpose is to use cudnnLSTM instead of regular  LSTM (layer 6), because regular LSTM works so slow (it uses 10% of GPU or less)

This code below works fine with tensorflow-gpu verison 1.1 however, I cannot use cudnnLSTM layer with tensorflow-gpu less thatn 1.2, so I updated my tensorflow-gpu to higher version, but now, I have the follolwing error. Is it a cudnn version issue?


_________________________________________________________________
Layer (type)                 [Output] Shape              Param #   
=================================================================
input_1 (InputLayer)         (None, 100, 20, 20, 1)    0         
_________________________________________________________________
time_distributed_1 (TimeDist (None, 100, 16, 16, 20)   520       
_________________________________________________________________
time_distributed_2 (TimeDist (None, 100, 5120)         0         
_________________________________________________________________
time_distributed_3 (TimeDist (None, 100, 1)            5121      
=================================================================
Total params: 5,641
Trainable params: 5,641
Non-trainable params: 0
_________________________________________________________________
Train on 30960 samples, validate on 7740 samples
Epoch 1/50
2018-04-10 10:40:15.085958: I T:\src\github\tensorflow\tensorflow\core\platform\cpu_feature_guard.cc:140] Your CPU supports instructions that this TensorFlow binary was not compiled to use: AVX2
2018-04-10 10:40:15.616355: I T:\src\github\tensorflow\tensorflow\core\common_runtime\gpu\gpu_device.cc:1344] Found device 0 with properties: 
name: GeForce GTX 1080 major: 6 minor: 1 memoryClockRate(GHz): 1.7335
pciBusID: 0000:02:00.0
totalMemory: 8.00GiB freeMemory: 6.64GiB
2018-04-10 10:40:15.616355: I T:\src\github\tensorflow\tensorflow\core\common_runtime\gpu\gpu_device.cc:1423] Adding visible gpu devices: 0
2018-04-10 10:40:16.911147: I T:\src\github\tensorflow\tensorflow\core\common_runtime\gpu\gpu_device.cc:911] Device interconnect StreamExecutor with strength 1 edge matrix:
2018-04-10 10:40:16.911147: I T:\src\github\tensorflow\tensorflow\core\common_runtime\gpu\gpu_device.cc:917]      0 
2018-04-10 10:40:16.911147: I T:\src\github\tensorflow\tensorflow\core\common_runtime\gpu\gpu_device.cc:930] 0:   N 
2018-04-10 10:40:16.911147: I T:\src\github\tensorflow\tensorflow\core\common_runtime\gpu\gpu_device.cc:1041] Created TensorFlow device (/job:localhost/replica:0/task:0/device:GPU:0 with 6411 MB memory) -> physical GPU (device: 0, name: GeForce GTX 1080, pci bus id: 0000:02:00.0, compute capability: 6.1)
2018-04-10 10:40:19.469530: E T:\src\github\tensorflow\tensorflow\stream_executor\cuda\cuda_dnn.cc:396] Loaded runtime CuDNN library: 7101 (compatibility version 7100) but source was compiled with 7003 (compatibility version 7000).  If using a binary install, upgrade your CuDNN library to match.  If building from sources, make sure the library loaded at runtime matches a compatible version specified during compile configuration.
2018-04-10 10:40:19.469530: F T:\src\github\tensorflow\tensorflow\core\kernels\conv_ops.cc:712] Check failed: stream->parent()->GetConvolveAlgorithms( conv_parameters.ShouldIncludeWinogradNonfusedAlgo<T>(), &algorithms) 

----------------------------------------------------------
I have the following settings
keras                     2.1.5                    py36_0
tensorflow                1.7.0                    py36_0    aaronzs
tensorflow-gpu            1.7.0                    py36_0    aaronzs
Windows 7
Cuda version 9.0
cudnn version 6 (cudnn64_7)
GPU NVIDIA GeForce GTX 1080 8 GB

My code:
    # Functional model definition
    inputs = Input(shape=(seqLength, 20, 20, 1))  # layer[0]
    output_of_layer1 = TimeDistributed(Conv2D(20, (5, 5), activation="relu"))(inputs)  # layer[1]
    output_of_layer2 = TimeDistributed(MaxPooling2D(pool_size=(2, 2)))(output_of_layer1)  # layer[2]
    output_of_layer3 = TimeDistributed(Conv2D(50, (5, 5), activation="relu"))(output_of_layer2)  # layer[3]
    output_of_layer4 = TimeDistributed(MaxPooling2D(pool_size=(2, 2)))(output_of_layer3)  # layer[4]
    output_of_layer5 = TimeDistributed(Flatten())(output_of_layer4)  # layer[5]
    output_of_layer6 = Bidirectional(CuDNNLSTM(100, return_sequences='true', unit_forget_bias=True),
                                     merge_mode='concat')(output_of_layer5)  # layer[6]
    # concatenated_outputs_as_input = Concatenate(axis=-1)([output_of_layer5, output_of_layer6])
    # output_of_layer7 = Bidirectional(LSTM(100, activation="sigmoid", return_sequences='true', unit_forget_bias = True), merge_mode = 'concat')(concatenated_outputs_as_input)# layer[7]
    output_of_layer8 = TimeDistributed(Dense(1, activation="sigmoid"))(output_of_layer6)  # leayer[8]
    model = Model(inputs=inputs, outputs=output_of_layer8)

    model.summary()
    nepochs = 50
    # learning_rate = 0.01
    # decay_rate = 0.00001 #learning_rate / nepochs
    sgd = optimizers.SGD(lr=0.01, decay=0, momentum=0.95, nesterov=False, clipvalue=20)
    model.compile(loss='binary_crossentropy', optimizer=sgd, metrics=['accuracy'])
    checkpoint = ModelCheckpoint('trained_plaqueDetection_model_bidirectional.h5', monitor='val_acc', verbose=1, save_best_only=False, mode='max')
    callbacks_list = [checkpoint]
    history = model.fit(training_dataset[0], training_dataset[1], epochs = nepochs, batch_size=10, shuffle=True, verbose = 1, validation_split = 0.2, callbacks = callbacks_list);
plot_losses(history)


Thanks!

**Comment by reedwm** [comment]
I apologize, but I am having a hard time understanding what the problem is, where the problem is, and what version it affects. Please resubmit and pay attention to the issue template (https://github.com/tensorflow/tensorflow/issues/new). Please provide all the information it asks. Thank you.
